content/en/docs/usage/ingress/ingress-istio.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Istio as Ingress"
+weight: 2
+description: "Learn how to use Istio as Ingress with Fission."
+---
+
+Istio is one of the most popular open source service mesh.
+It provides a platform to manage microservices and also facilitates communication between them.
+It offers features for traffic management, security, observability and extensibility.
+In this document, we shall look into how you can use Istio as Ingress with Fission.
+
+## Pre-requisites
+
+To begin with, you need to have Istio installed on your cluster.
+There are multiple options to set up Istio and based on your setup, you can choose one of many ways available to [install Istio](https://istio.io/latest/docs/setup/install/).
+
+To install Fission you can refer to [Fission Installation](/docs/installation).
+Set the `enableIstio` flag as `true` during the setup to enable Istio in Fission
+
+```bash
+helm install --version v1.15.1 --namespace fission \
```

Comment:
Use `{{% release-version %}}` instead of `v1.15.1` so that latest version gets rendered automatically.

Same we use in installation docs https://github.com/fission/fission.io/blob/master/content/en/docs/installation/_index.en.md